airbyte-integrations/bases/connector-acceptance-test/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 0.7.2
+TestConnection: assert that a check with `exception` status emits a trace message.
+
```

Comment:
Wondering if this should be bumped differently, as it's a new test we expect some to fail 🤔

Reply:
Only two connectors are using `status: "exception"` in their acceptance test config (`dixa` and `file`). They both are not passing CAT at the moment.
I don't expect additional connectors to fail as only these connectors + airtable will go into the `inputs.status == ConnectionTestConfig.Status.Exception:` statement.